src/lib/rest/clientSocketHttp.cpp

```diff
@@ -291,8 +292,15 @@ std::string sendHttpSocket
   }
 
   // ----- Accept
-  headers = curl_slist_append(headers, "Accept: application/xml, application/json");
-  outgoingMsgSize += 41; // from "Accept: application/xml, application/json"
+  std::string acceptedFormats = "application/xml, application/json";
```

Comment:
Is there actually any "functional" change in this part? Or the modification is just to make things cleaner? (I guess is the second, but I'm not fully sure).